src/ui/layout/tree-item.vue

```diff
@@ -1,8 +1,9 @@
 <template>
 <div
     :style="{
```

Comment:
should be a computed style object

Reply:
Cleaned this up a bit and was able to remove itemTop computed variable as well.